scripts/extract-supabase-service-key.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Execute the supabase status command and capture its output
+STATUS_OUTPUT=$(pnpm --filter @liam-hq/db exec supabase status)
```

Comment:
nits 

`pnpm supabase status --output json` might be useful for that purpose. But  in this PR for now, this sh is ok.